ui/viewers/nuxeo-image-viewer.js

```diff
@@ -78,6 +78,14 @@ import '../nuxeo-icons.js';
           value: false,
         },
 
+        /**
+         * A short text alternative for the image.
+         */
+        alt: {
+          type: String,
+          value: null,
```

Comment:
would it make sense for the default to be something like "Image" instead of null?

Reply:
also this should be part of next commit correct? :)

Reply:
> would it make sense for the default to be something like "Image" instead of null?
It would be redundant since the screen reader will mention the tag `image`.

> also this should be part of next commit correct? :)
that is a good question that crossed my mind, I'm ok to change it. My logic was something like this: the first commit adds an alt attribute to wherever it is needed; the second one adds a unit test to a specific propriety. I will see if anyone else has also a strong feeling about this

Reply:
> > would it make sense for the default to be something like "Image" instead of null?
> 
> It would be redundant since the screen reader will mention the tag `image`.
> 
> > also this should be part of next commit correct? :)
> 
> that is a good question that crossed my mind, I'm ok to change it. My logic was something like this: the first commit adds an alt attribute to wherever it is needed; the second one adds a unit test to a specific propriety. I will see if anyone else has also a strong feeling about this

For me it makes sense to separate the commit that adds the attribute, and the one with the unit test.